.dotnet/src/Custom/Moderations/ModerationResult.cs

```diff
@@ -8,19 +8,22 @@ namespace OpenAI.Moderations;
 [CodeGenModel("CreateModerationResponseResult")]
 [CodeGenSuppress("Categories")]
 [CodeGenSuppress("CategoryScores")]
-[CodeGenSuppress(nameof(ModerationResult), typeof(bool), typeof(InternalModerationCategories), typeof(InternalModerationCategoryScores))]
-[CodeGenSuppress(nameof(ModerationResult), typeof(bool), typeof(InternalModerationCategories), typeof(InternalModerationCategoryScores), typeof(IDictionary<string, BinaryData>))]
+[CodeGenSuppress("CategoryAppliedInputTypes")]
```

Comment:
We need to add new methods to `ModerationClient` that can take the multimodal inputs as parameters. Should we do it here or in a separate PR?

Reply:
Let's do it in a separate one. I added an item to the internal repository board.